\chapter{Objectives}

\shorthandoff{-} 

Thanks to the collection of environmental \tax{E. coli} isolates \cite{ishii2006presence}, we have access to 480 various strains thorough the whole phylogenetic tree of this species.
We will use these strains in order to elucidate how natural selection acts on transcriptional responses and epigenetic memory of several promoters by investigating both genotypic and phenotypic variation in them.

\section{Explore genotypic and phenotypic variation in promoters among environmental \tax{E. coli} isolates.}
%%% I'M NOT SURE HOW TO SPECIFY WHERE THE PRELIMINARY DATA WERE COLLECTED, SO I'M OPENED TO SUGGESTIONS
Some preliminary data were already collected in my supervisor's previous laboratory using \tax{lacZ} promoter.
They identified differences in the promoter sequence and further downstream in \tax{lacZ} gene itself in natural isolate SC1\textunderscore D9 when compared to a classic laboratory strain MG1655.
Differential fluorescence was observed when these promoters were cloned upstream of GFP gene on a plasmid and induced by IPTG.
And not only the sequence itself, but also genetic background seems to affect the fluorescence levels and thus \tax{lacZ} promoter activity.
I will follow up on and expand these findings with \tax{lacZ} promoter and include other promoters and strains as well.
The specific steps of this objective are described further below.

\subsection{Choose promoter sequences among environmental \tax{E. coli} isolates to study}
For the beginning we chose well studied \tax{lacZ} and \tax{recA} promoters, because each create a different type of cell response - carbon source catabolism and stress response, respectively.
We would like to include other promoters as well.
For this, promoters with known function and regulation mechanism will be picked based on annotated MG1655 genome.
Then we will search for corresponding sequences in genomes of natural isolates and identify the sequence differences in them.
As we want to cover a broad variation scale on genetic level we will select promoter sequences which are both highly and slightly diverse among natural isolates.
Those which are mainly bound by RNA polymerase with a conventional $\sigma^{70}$ factor will be preferred.

\subsection{Investigate how the genotypic variation relates to the phenotypic variation}
Selected promoters from different strains will be cloned upstream of a fluorescent reporter and then integrated into chromosome of those strains.
Variation in promoter activity will be then monitored under sets of conditions to determine the dynamics in studied transcriptional responses.
For this purposes flow cytometry will be mainly used.
However, we want to investigate these trait at single cell level as well, so fluorescent microscopy and microfluidics with high throughput image data analysis will follow.
Microfluidics can also help us to go deeper into transcriptional regulation under fluctuating environments and potential epigenetic memory.


\section{Compare natural variation in transcriptional responses with random distribution.}
Results from the previous aim will give us some view on how strong selection acts on the studied promoters in means of mean expression, sensitivity, plasticity and noise.
But to get more detailed picture, observed variation needs to be compared to a random variation - i.e. variation expected to occur if there is no selection acting on the particular trait of promoter.
This will be achieved by a random mutagenesis and analysis of variation produced this way as delineated below.

\subsection{Generate libraries of promoters by random mutagenesis}
First, we need to create a random variation in the selected promoters.
Error-prone polymerases will be used for that and produced variants will be then cloned upstream of a fluorescent reporters and incorporated into chromosome of all studied genotypic backgrounds the same way as for the naturally occurring promoters.

\subsection{Investigate phenotypic variation in promoters created by random mutagenesis}
Next, we will compare the observed phenotypic variation in native promoters to those randomly generated among studied strains using the same techniques as previously.
Based on those results we will be able to define in which direction the selection acts on particular promoters compared to the random distribution.
We should be also able to answer a question whether the chosen promoters were selected for a certain level of sensitivity, noise or memory and if so, which one.


\shorthandon{-} 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
