\chapter*{Objectives}
\addcontentsline{toc}{chapter}{Objectives}

\shorthandoff{-} 

Thanks to the collection of environmental \tax{E. coli} strain isolated near Lake Superior, MN, USA \cite{ishii2006presence}, we have access to 480 various strains thorough the whole phylogenetic tree of this species.
We will use these strains in order to elucidate how natural selection acts on transcriptional responses and epigenetic memory of several promoters by investigating both genotypic and phenotypic variation in them.

\begin{enumerate}[font=\bfseries]

	\item \textbf{Quantify genotypic and phenotypic variation in promoters among environmental \tax{E. coli} isolates}
	
	In order to quantify variation in transcriptional, we will focus initially on studying natural variation in the response of the \tax{lac} operon in the presence of different carbon sources, either separately or in combination.
	To begin we will study these responses using a promoter-based system.
	Previously, initial data were collected which suggested that differences in the \tax{lac} promoter sequence and further downstream in \tax{lacZ} gene itself result in differential gene expression.
	Moreover not only the sequence of the promoter, but also the genetic background of the strain was observed to affect the level of expression (i.e. cis- and trans- effects).
	I will follow up on and expand these findings with \tax{lacZ} promoter and include other promoters and strains as well.
	The specific steps of this objective are described further below.

	\begin{enumerate}[font=\bfseries]
	
		\item \textbf{Choose promoter sequences among environmental \tax{E. coli} isolates to study}
		
		For the beginning we have selected the well studied \tax{lacZ} and \tax{recA} promoters, because each create a different type of cell response - carbon source catabolism and stress response, respectively.
		We will select three(\textcolor{red}{?}) other transcriptional circuits to study as well.
		Promoters with known function and regulation mechanism will be picked based on annotated MG1655 genome.
		Then we will search for corresponding sequences in genomes of natural isolates and identify the sequence differences in them.
		As we want to cover a broad variation scale on genetic level we will select promoter sequences which are both highly and slightly diverse among natural isolates.
		Those which are mainly bound by RNA polymerase with a conventional $\sigma^{70}$ factor will be preferred.

		\item \textbf{Quantify phenotypic variation in promoters using plasmid system}
		
		% 17.9.2018
		% maybe let's say as a first step to look for differences in responses we will study the plasmid system using the (already clones) K12 sequence. I think this makes sense.


		\item \textbf{Confirm observed relationships between genotypic and phenotypic variation by chromosomal integration}
		
		As plasmid based models might cause problems in interpreting the results in respect to usually single copy of promoter in bacterial chromosome and multiple copies of plasmid, we will clone a subset of promoters showing differential expression upstream of a fluorescent reporter and then integrate them into chromosome of studied strains.
		We will then monitor variation under sets of conditions to determine the dynamics in studied transcriptional responses.
		For this purposes flow cytometry will be mainly used.
		However, we want to investigate these trait not only at single cell level, but under dynamic conditions as well, so fluorescent microscopy and microfluidics with high throughput image data analysis will follow.
		Microfluidics can also help us to go deeper into transcriptional regulation under fluctuating environments and potential epigenetic memory.

	\end{enumerate}
	
	\item \textbf{Compare natural variation in transcriptional responses with random distribution}
	
	Results from the previous aim will give us some view on how much natural variation there is in expression, sensitivity, plasticity and noise.
	To understand whether selection has acted to increase or decrease such variation, we need a neutral model to compare to.
	Thus, we will generate sets of promoters with random mutations - i.e. variation expected to occur if there is no selection acting on the particular trait of promoter.
	This will be achieved by a random mutagenesis and analysis of variation produced this way as delineated below.

	\begin{enumerate}[font=\bfseries]
	
		\item \textbf{Generate libraries of promoters by random mutagenesis}
		
		First, we need to create a random variation in the selected promoters.
		% 17.9.2018
		% perhaps a bit more detail, e.g. avg number of changes per promoter, number of variants (will you study 10 or 100 or 1000)...
		Error-prone polymerases will be used for that and produced variants will be then cloned upstream of a fluorescent reporters and incorporated into chromosome of all studied genotypic backgrounds the same way as for the naturally occurring promoters.

		\item \textbf{Investigate phenotypic variation in promoters created by random mutagenesis}
		
		% 17.9.2018
		% "same techniques as previously" - again more detail. I guess you can study hundreds using cytometry but only select a subset to study with microscopy. You could say this.
		Next, we will compare the observed phenotypic variation in native promoters to those randomly generated among studied strains using the same techniques as previously.
		% 17.9.2018
		% be more direct: "whether selection has acted to decrease or increase..."
		Based on those results we will be able to define in which direction the selection acts on particular promoters compared to the random distribution.
		We should be also able to answer a question whether the chosen promoters were selected for a certain level of sensitivity, noise or memory and if so, which one.

	
	\end{enumerate}

\end{enumerate}


\shorthandon{-} 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
