\chapter{Materials and Methods}
%\setcounter{page}{1}
%\pagenumbering{arabic}
%\addcontentsline{toc}{chapter}{Metody}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%% GENERUJ TEXT %%%%%%%%%%%%%

\shorthandoff{-} 

\section{Strains and growth conditions}
All strains used in this work belong to the genus \tax{Escherichia coli} and are mainly derived from a classical laboratory strain K12 (MG1655) and an environmental strain SC1\textunderscore D9 isolated from a bank of St. Louis River in Minnesota within a great sampling study \cite{ishii2006presence}.
In future work additional environmental isolates are likely to be used.

\begin{center}
	\begin{longtable}[c]{|l|c|c|c|c|}
\caption{Strains used in the study so far} \label{strains} \\

\toprule \multicolumn{1}{|l|}{\textbf{Strain ID}} & \multicolumn{1}{c|}{\textbf{Relevant genotype}} & \multicolumn{1}{c|}{\textbf{Promoter}} & \multicolumn{1}{c|}{\textbf{Ancestor}} & \multicolumn{1}{c|}{\textbf{Reference}} \\
\midrule
\endhead

\bottomrule
\endlastfoot

MG1655 & \tax{E. coli} K12 wild-type & - & - & \cite{blattner1997complete} \\
\hline
SC1\textunderscore D9 & \tax{E. coli} natural isolate & - & - & \cite{ishii2006presence} \\
\hline
pUA66-K12 & pUA66 & - & MG1655 & \cite{zaslaver2006comprehensive} \\
\hline
pU139-K12 & pU139 & - & MG1655 & \cite{zaslaver2006comprehensive} \\
\hline
pU139-D9 & pU139 & - & SC1\textunderscore D9 & this study \\
\hline
ASC662 & \tax{lacZ-GFP} & - & MG1655 & \cite{kiviet2014stochasticity} \\
\hline
pZ1\textunderscore K12-K12 & \tax{placZ::GFP} & MG1655 & MG1655 & \cite{zaslaver2006comprehensive} \\
\hline
pZ2\textunderscore K12-K12 & \tax{placZ::GFP} & MG1655 & MG1655 & this study \\
\hline
pZ2\textunderscore K12-D9 & \tax{placZ::GFP} & MG1655 & SC1\textunderscore D9 & this study \\
\hline
pZ\textunderscore D9-D9 & \tax{placZ::GFP} & SC1\textunderscore D9 & SC1\textunderscore D9 & this study \\
\hline
pZ\textunderscore D9-K12 & \tax{placZ::GFP} & SC1\textunderscore D9 & MG1655 & this study \\
\hline
pZm1\textunderscore D9-K12 & \tax{placZm172::GFP} & SC1\textunderscore D9 & MG1655 & this study \\
\hline
pZm2\textunderscore D9-K12 & \tax{placZm279::GFP} & SC1\textunderscore D9 & MG1655 & this study \\
\hline
pA\textunderscore K12-K12 & \tax{precA::GFP} & MG1655 & MG1655 & \cite{zaslaver2006comprehensive} \\
\hline
pA\textunderscore K12-D9 & \tax{precA::GFP} & MG1655 & SC1\textunderscore D9 & this study \\
\hline
Top10 & \text{*} & - & MG1655 & [Invitrogen] \\
\hline
pLW001-Top10 & pLW001 & - & Top10 & this study \\
	\end{longtable}
\footnotesize
	\emph{\text{*}} Fâ€“ mcrA $\Delta$(mrr-hsdRMS-mcrBC) $\Phi$80lacZ$\Delta$M15 $\Delta$lacX74 recA1 araD139 $\Delta$(ara leu)7697 galU galK rpsL(Str$^{R}$) endA1 nupG\\*
\end{center}

For flow cytometry and microscopy purposes M9 minimal medium supplemented with 0.4 \% of a single carbon source was used.
As those D(+)-glucose, lactose, D(+)-galactose, L-arabinose and D(-)-ribose were chosen.
In other cases in LB medium (1.5~\%~agar or liquid) was used.
Kanamycin to concentration 50 $\mu$g/ml was also added to the media if the bacterium contained pU139 or pUA66 plasmid (with or without promoter upstream of GFP).
All strains were grown at 37$^{\circ}$C, except for pLW001-Top10 which was grown at 30$^{\circ}$C (and in presence of 100 $\mu$g/ml Ampicillin as well).


\section{DNA isolation, PCR and sequencing}
For sequencing and cloning purposes 0.5 ml of LB supplemented with an appropriate antibiotic was inoculated by a single colony of desired strain and grown for 8 h at 37$^{\circ}$C or for 16 h at 30$^{\circ}$C with shaking.
5 $\mu$l of this culture was transferred into 100 $\mu$l of sterile deionized water and incubated at 95$^{\circ}$C for 5 min.
Isolated DNA was stored at -20$^{\circ}$C freezer.
For screening of positive clones a colony PCR was performed - I touched a single colony by a sterile pipette tip and swirled it in PCR tube with all reagents ready.

In case of screening DreamTaq Green PCR Master Mix (2X) was used for PCR according to manufacturer's \href{https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0012704_DreamTaq_Green_PCR_MasterMix_K1081_UG.pdf}{instructions}.
The PCR was run at these conditions: 3~min @ 95$^{\circ}$C; 35 cycles: 30~s @ 94$^{\circ}$C + 30~s @ T$_{a}$ + 60~s/kb @ 72$^{\circ}$C; 5~min @ 72$^{\circ}$C.
For sequencing PCR with recombinant \tax{Taq} DNA polymerase [Invitrogen] was performed according to manufacturer's \href{https://assets.thermofisher.com/TFS-Assets/LSG/manuals/0814_Taq_DNA_Polymerase_recombinant.pdf}{instructions}.
PCR conditions were as follows: 3~min @ 95$^{\circ}$C; 35 cycles: 45~s @ 94$^{\circ}$C + 30~s @ T$_{a}$ + 90~s/kb @ 72$^{\circ}$C; 10~min @ 72$^{\circ}$C. Used T$_{a}$ for different primers are listed in Table \ref{pcr}.

\begin{center}
	\begin{longtable}[c]{|l|c|c|c|}
\caption{List of primers} \label{pcr} \\

\toprule \multicolumn{1}{|l|}{\textbf{Primer}} & \multicolumn{1}{c|}{\textbf{Sequence 5' -- 3'}} & \multicolumn{1}{c|}{\textbf{T$_{a}$}} & \multicolumn{1}{c|}{\textbf{Purpose}} \\
\midrule
\endhead

\bottomrule
\endlastfoot

pUA66\textunderscore insert\textunderscore R & \footnotesize{\texttt{TCGCAAAGCATTGAAGACCATACGC}} & \multirow{2}{*}{56$^{\circ}$C} & \multirow{2}{*}{sequencing} \\
pUA66\textunderscore insert\textunderscore F & \footnotesize{\texttt{TTGTCTGTTGTGCCCAGTCATAGC}} & & \\
\hline
pUA66\textunderscore clon\textunderscore R & \footnotesize{\texttt{CGCATAGGGCCCGGATTTGTCCTACTCAGGAGAGCG}} & \multirow{2}{*}{54$^{\circ}$C} & \multirow{2}{*}{cloning} \\
pUA66\textunderscore clon\textunderscore F & \footnotesize{\texttt{TAAGGTCCCGGGCTGTCTCTTGATCAGATCTTGATCCCC}} & & \\
\hline
ATT-RP2 & \footnotesize{\texttt{CAGAATCCCTGCTTCGTCCA}} & \multirow{2}{*}{55$^{\circ}$C} & \multirow{2}{*}{sequencing} \\
PSC-FP2 & \footnotesize{\texttt{TATCGAATCAAAGCTGCCGA}} & & \\
	\end{longtable}
\end{center}

Desired PCR products were column purified before dispatch for Illumina sequencing to Macrogen, Korea.
Column purification was performed using E.Z.N.A.\textsuperscript{\textregistered} Cycle Pure Kit according to manufacturer's \href{http://omegabiotek.com/store/wp-content/uploads/2013/09/D6492_D6493-Cycle-Pure-Kit-Combo-Online.pdf}{centrifugation protocol}.
Purified DNA was eluted into 30 $\mu$l of sterile deionized water.

\section{Cloning}
\subsection{TOPO TA cloning}
Desired insert was amplified from pUA66-K12 strain using pCR\textsuperscript{TM}8/GW/TOPO\textsuperscript{\textregistered} reagents according to manufacturer's \href{https://assets.thermofisher.com/TFS-Assets/LSG/manuals/pcr8gwtopo_man.pdf}{instructions}, except for recombinant \tax{Taq} DNA polymerase [Invitrogen] and primers (see Table \ref{pcr}).
%%% SHOULD I ADD pUA66 PLASMID SCHEME TO APPENDIX?
PCR was run as follows: 3~min @ 95$^{\circ}$C; 35~cycles: 45~s @ 95$^{\circ}$C + 30~s @ T$_{a}$ + 90~s/kb @ 72$^{\circ}$C; 30~min @ 72$^{\circ}$C.
20 $\mu$l of PCR product was then column purified using E.Z.N.A.\textsuperscript{\textregistered} Cycle Pure Kit according to manufacturer's \href{http://omegabiotek.com/store/wp-content/uploads/2013/09/D6492_D6493-Cycle-Pure-Kit-Combo-Online.pdf}{centrifugation protocol}.
Purified DNA was eluted into 30 $\mu$l of sterile deionized water.

TOPO\textsuperscript{\textregistered} Cloning reaction was performed according to manufacturer's \href{https://assets.thermofisher.com/TFS-Assets/LSG/manuals/pcr8gwtopo_man.pdf}{instructions} with 4 $\mu$l of purified PCR product.
The reaction was incubated at 22$^{\circ}$C for 30~min and stored in -20$^{\circ}$C freezer before electroporation or One Shot\textsuperscript{\textregistered} Chemical transformation.
In the case of using chemical transformation it was performed according to manufacturer's \href{https://assets.thermofisher.com/TFS-Assets/LSG/manuals/pcr8gwtopo_man.pdf}{instructions} and using their S.O.C. medium.

\subsection{Construction of pADOUCH plasmid}
First pLW001 vector and TOPO vector with desired insert (see TOPO TA cloning) were isolated from donor strains grown in liquid LB supplemented with an appropriate antibiotic.
This was performed using StrataPrep Plasmid Miniprep Kit according to manufacturer's \href{https://www.agilent.com/cs/library/usermanuals/public/400766.pdf}{instructions}.
Changes were made in using sterile deionized water instead of elution buffer and growing pLW001-Top10 donor strain for 24 h at 30$^{\circ}$C prior to the very plasmid isolation.
Restriction digestion was then set up using ApaI and XmaI restriction enzymes in final volume of 50 $\mu$l with incubation at 25$^{\circ}$C for 2 h followed by 37$^{\circ}$C for another 2~h.
Digested pLW001 vector was then dephosphorylated with rSAP treatment at 37$^{\circ}$C for 30~min and next both reactions were heat inactivated at 65$^{\circ}$C for 30 min.

Digested vector and insert were gel extracted using StrataPrep DNA Gel Extraction Kit according to manufacturer's \href{https://www.agilent.com/cs/library/usermanuals/public/400766.pdf}{instructions} with the exception that 50 $\mu$l of sterile deionized water was used for elution instead of elution buffer.
Overnight ligation at 16$^{\circ}$C was then performed using T4 ligase in final volume of 20 $\mu$l and keeping DNA concentration between 1-10 ng/$\mu$l.
Ligation reaction was used directly for electroporation or stored -20$^{\circ}$C freezer before use.

\section{Transformation}
Desired plasmids for transformation were isolated from donor strains grown in liquid LB supplemented with appropriated antibiotic and using StrataPrep Plasmid Miniprep Kit according to manufacturer's \href{https://www.agilent.com/cs/library/usermanuals/public/400766.pdf}{instructions}.
The only exceptions were using sterile deionized water instead of elution buffer and growing pLW001-Top10 strain for 24 h at 30$^{\circ}$C prior to the very plasmid isolation.
If the concentration of acquired plasmid DNA was too low, it was concentrated using centrifugal vacuum concentrator [Eppendorf\textsuperscript{\textregistered} Basic Model 5301] at room temperature.
In case of transformation by ligation mixture, this was used directly without any purification.

\subsection{TSS transformation}
A flask of 100~ml capacity containing 10~ml of liquid LB was inoculated by 0.1~ml of overnight MG1655 culture (for other strains or plasmid size above 5~kb electroporation was used) and incubated at 37$^{\circ}$C with shaking.
When OD$_{600}$ rose to values between 0.5 -- 0.9 the culture was fast chilled by swirling in ice slurry for 15~min.
Next 0.75~ml of the chilled culture was transferred into 5~ml tube containing the same amount of ice cold 2x TSS (see Table \ref{tss}) and was incubated on ice slurry for 30~min.
2~$\mu$l of desired plasmid was then added to the cells with subsequent incubation on ice slurry for another 1~h.
After additional incubation for 1~h at 37$^{\circ}$C with shaking, cells were centrifuged at maximum speed (i.e. 14000~rpm = 20817~G) for 5~min at room temperature, supernatant was discarded and pellet re-suspended in 0.2~ml of LB.
From this 10-fold dilutions were prepared up to 10$^{-3}$ and these were spread by sterile beads on LB agar plates containing appropriate antibiotic.
Non-diluted culture was plated as well.
Plates were then incubated overnight at 37$^{\circ}$C and single colonies were checked for desired plasmid presence by PCR.

%\newpage

\begin{center}
	\begin{longtable}[c]{|l|c|c|}
\caption{Content of 2x TSS} \label{tss} \\

\toprule \multicolumn{1}{|l|}{\textbf{Component}} & \multicolumn{1}{c|}{\textbf{Amount per 100~ml of 2x TSS}} \\
\midrule
\endhead

\bottomrule
\endlastfoot

Bacto-Tryptone & 0.8~g \\
\hline
Yeast extract & 0.5~g \\
\hline
NaCl & 0.5~g \\
\hline
PEG 8000 & 20.0~g \\
\hline
1M MgSO$_{4}$ & 10~ml \\
\hline
DMSO & 10~ml \\
	\end{longtable}
\end{center}

\subsection{Preparation of electrocompetent cells}
A flask of 1 l capacity containing 100 ml of liquid LB was inoculated by 1 ml of overnight culture and incubated at 37$^{\circ}$C with shaking.
After reaching OD$_{600}$ between 0.5 -- 0.9 the culture was fast chilled by swirling in ice slurry for 5 -- 10 min and then kept in the slurry for 1 h.
Next the culture was transferred into 2 pre-chilled 50 ml falcon tubes and centrifuged  at 4$^{\circ}$C for 10 min at 4200 G.
The supernatant was discarded and pellet was washed twice in about 25 ml of 10 \% glycerol with centrifugation at 4$^{\circ}$C for 10 min at 4500 G.
The final supernatant (about 0.5 ml) was pooled into one falcon tube and aliquoted into pre-chilled (in -80$^{\circ}$C freezer) 1.7 ml tubes.

\subsection{Electroporation}
A tube of electrocompetent cells was thawed on ice and 1 $\mu$l of desired plasmid or 3 $\mu$l of ligation mix was added to it.
The cells were then transferred into 0.1 cm electroporation cuvette [BioRad] and electroporated at 1850 V (or 1850 V if vector was bigger than 10 kb) [Eppendorf Eporator\textsuperscript{\textregistered}].
500 ml of S.O.C. medium was added immediately to the cuvette and the whole content then transferred into pre-warmed 5 ml tube.
After incubation for 1~h at 37$^{\circ}$C (or 2 h at 30$^{\circ}$C in case of pLW001 plasmid background) with shaking, 10-fold dilutions up to 10$^{-3}$ were prepared and spread by sterile beads on LB agar plates containing an appropriate antibiotic.
Non-diluted culture was plated as well.
Plates were then incubated overnight at 37$^{\circ}$C (or two overnights at 30$^{\circ}$C) and single colonies were checked for desired plasmid presence by colony PCR.


\section{Activity assays of promoters}
\subsection{Promoter \tax{lacZ}}
M9 media with 0.4~\% of chosen single carbon sources (0.2~ml) in 96-well micro-titre plates were inoculated by strains from a glycerol stock in duplicates using a pin replicator.
After overnight incubation at 37$^{\circ}$C without shaking the strains were transferred into the same fresh media by the same pin replicator and incubated again at 37$^{\circ}$C without shaking for 4~h.
Before acquiring the data from samples with flow cytometer [BD FACSDiva version 6.1.3], all strains were 10-fold (MG1655 background) or 20-fold (SC1\textunderscore D9 background) diluted in 1x~PBS with $\sim$1~\% formaldehyde.
Data from 10 -- 100 thousands of events were collected using GFP filter 513/17 [BD] and exported into Flow Cytometry Standard (FCS) files.
These files were then analysed in R (version 3.4.3) using flowCore package (version 1.44.1) \cite{hahne2009flowcore}.
Scripts used for data analysis are available at \href{https://github.com/marketavlkova/LacZ_FC}{GitHub}.

\subsection{Promoter \tax{recA}}
M9 media supplemented with 0.4~\% glucose and Ciprofloxacin at sub-MIC concentrations (0.5~ng/ml, 1.0~ng/ml, 2.0~ng/ml and 4.0~ng/ml) in 96-well micro-titre plates were inoculated by strains from a glycerol stock in duplicates using a pin replicator.
Further steps were the same as for \tax{lacZ} promoter assay (using M9 with glucose and Ciprofloxacin).
Scripts used for data analysis of \tax{recA} promoter are available at \href{https://github.com/marketavlkova/RecA}{GitHub} in a separated repository.


\section{Microscopy}
For fluorescent microscopy 1~$\mu$l of fresh 4~h culture (see promoter activity assay for growth conditions) was transferred to 1~\% agarose pad prepared from the same media the cells were grown in.
Multiple agarose pads were placed within a single gene frame on a microscopy slide and after adding cultures covered with a coverslip.
Two positions per an agarose pad were chosen and automated time-lapse phase contrast and fluorescent (GFP; 200~ms exposure time) microscopy was set-up.
Both phase contrast and fluorescent images were taken using 100x immersion objectives and Hamamatsu camera [ORCA-Flash4.0].
All pictures were stored and handled in TIFF format.

\shorthandon{-} 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
