\chapter*{Summary and future work}
\addcontentsline{toc}{chapter}{Summary}

\shorthandoff{-}
Although much effort and focus have been dedicated to the study of natural selection, a lot still remains unclear in this field.
Recently many studies shown that even in clonal bacterial populations we can see different cell responses to the same stimulus.
This study aims to shed more light on whether the mechanisms leading to such variations within a population with the same genetic background are selected for and if so, whether in order to decrease or increase variability in the particular mechanism (i.e., transcriptional noise, plasticity, response speed, sensitivity, and epigenetic memory).
For this the best known bacterial model organism \tax{E. coli} will be used, however, not so well studied environmental isolates of this species will constitute the crucial part in understanding what role the natural selection plays.
The main objectives of this work are 1. Quantify genotypic and phenotypic variation in promoters among environmental \tax{E. coli} isolates and 2. Compare natural variation in transcriptional responses with a neutral model.

\bigbreak
Result summary of this study in progress:
\begin{enumerate}[font=\bfseries]

    \item \textbf{Quantify genotypic and phenotypic variation in promoters among environmental \tax{E. coli} isolates}
    
    \begin{itemize}
    
        \item well-understood \tax{lacZ} and \tax{recA} promoters were used so far to establish main workflows of this study (i.e., flow cytometry and fluorescent microscopy) on a common lab strain MG1655 and one environmental strain SC1\textunderscore D9
        \item a set of five carbon sources, including lactose, was chosen to study \tax{lacZ} promoter responses both separately and in combination
        \item a range of various sub-lethal Ciprofloxacin concentrations was chosen for \tax{recA} promoter study
        \item differences in SC1\textunderscore D9 and MG1655 \tax{lacZ} promoter sequence lead to differential GFP expression in the used plasmid-based model
        \item \tax{lacZ} promoter activity is affected not only by the sequence itself but by the genetic background of the strain as well
        \item \tax{lacZ} promoter plasmid-based model is sensitive to the non-promoter sequences included as a part of the promoter insert upstream of GFP gene
        \item chromosomal integration of GFP gives different expression patterns in non-lactose environments in the case of \tax{lacZ} promoter experiments
        \item \tax{recA} promoter activity increases with increasing Ciprofloxacin concentration
        \item similarly to \tax{lacZ} promoter, the genetic background plays a role in \tax{recA} promoter activity besides the nucleotide sequence itself
        \item fluorescence values obtained by flow cytometry under the exposure of 4.0 ng/ml of Ciprofloxacin are distorted, probably by cell filamentation
        \item plasmid for chromosomal integration of the \tax{promoter::GFP} sequences was constructed
    
    \end{itemize}

\end{enumerate} 

Summary of future work associated with this study:

\begin{enumerate}

    \item \textbf{Quantify genotypic and phenotypic variation in promoters among environmental \tax{E. coli} isolates}
    
    \begin{itemize}
    
        \item 3 more promoters and sets of conditions to study them under are to be defined
        \item about 10 more strains remain to be selected for further work
        \item \tax{placZ::GFP} sequences are to be integrated into a chromosome of studied strains using constructed pADOUCH plasmid
        \item incorporation of a sequence coding an enzymatic auto-splicing RNA downstream of promoter inserts is being considered to avoid differences in 5' end sequences of GFP gene's mRNAs
        \item consistency in GFP expression triggered by \tax{recA} promoter between plasmid-based model and chromosomal integration is to be checked
        \item software for cell segmentation and tracking in phase contrast and/or fluorescent microscopy images remains to be selected
        \item microfluidics setups and workflows are to be established
        \item occurrence of epigenetic memory in selected promoter networks is to be investigated
            
    \end{itemize}
    
    \item \textbf{Compare natural variation in transcriptional responses with neutral model}
    
    \begin{itemize}
    
        \item libraries of selected promoter variants are to be generated by random mutagenesis
        \item obtained variants of promoters from the libraries are to be cloned into studied strains using a plasmid-based system or chromosomal integration with a fluorescent reporter
        \item variations in generated promoter variants are to be measured among studied strains using flow cytometry
        \item differences in generated promoter variants behaviour are to be investigated under dynamic and fluctuating conditions by fluorescent microscopy and microfluidics
        \item natural promoter variations are to be compared to the values obtained from neutral models determining whether natural selection has acted to increase or decrease transcriptional noise, plasticity, response sensitivity, speed and memory
    
    \end{itemize}

\end{enumerate}

Here I have reviewed the literature relevant to my confirmation proposal and outlined current research gaps this work will help to fill.
In the preliminary work I have shown that the main workflows using flow cytometry and fluorescent microscopy are feasible.
Moreover, we already collected data that give us a new information about the activity of the \tax{lacZ} and \tax{recA} promoters in various conditions and different genetic backgrounds.
This forms the basis of future thesis work.

\cleardoublepage%%% keeps correct headings

\shorthandon{-}
